src/schemas.py

```diff
@@ -59,79 +59,109 @@ class Distribution(BaseModel):
     media_type: str
     table_schema: TableSchema
 
-
-# If we wanted to provide the ability to attach arbitrary RDF we might want to
-# look at https://github.com/pydantic/pydantic/discussions/5853
-class Dataset(BaseModel):
-    context: Literal["https://data.ons.gov.uk/ns#"] = Field(alias="@context")
+class Edition(BaseModel):
     id: str = Field(alias="@id")
-    type: Literal["dcat:DatasetSeries"] = Field(alias="@type")
+    type: Literal["dcat:Dataset"] = Field(alias="@type")
+    in_series: str
     identifier: str
     title: str = Field(max_length=90)
     summary: str = Field(max_length=200)
     description: str = Field(max_length=250)
-    release_date: str = Field(
-        pattern=r"^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(Z|[+-]\d{2}:\d{2})?$"
-    )
-    next_release: str = Field(
-        pattern=r"^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(Z|[+-]\d{2}:\d{2})?$",
-    )
     publisher: str
     creator: str
     contact_point: ContactPoint
-    theme: Union[str, List[str]]
+    topics: Union[str, List[str]]
```

Comment:
Should we revert to `theme`? I would like to see our internal stuff directly aligned to the namespace, predicates, and objects we're employing and only through `@context` or the front end do we relabel things. Otherwise It's going to get way too confusing.

Reply:
so yeah I'm with you on that....but.. at the moment the stubbed jsonld responses we're using to power the api are also our source of data for loading in. So it has to be topics 😩 

conceptually no issues with keeping internal more rdf-y, just cant do that until we've progressed a ways and are looking at proper _distinct_ source files and no stub trickery.